tamr_unify_client/client.py

```diff
@@ -23,6 +23,18 @@ def successful(self):
 Response.successful = successful
 
 
+def fix_base_path(base_path):
```

Comment:
This is a short function, used only once.
This code should be inlined where used.

Reply:
done